test/lib/mocks/mocks-jsonwire.yaml

```diff
@@ -233,7 +233,7 @@ mocks:
     sessionId: *sessionId
     status: 0
 
-- url: '/wd/hub/session/1352110219202/log'
+- url: '/wd/hub/session/1352110219202/se/log'
```

Comment:
is this a valid endpoint?

Reply:
yes, the `/se/` is namespacing it purely for Selenium and showing that it is not in the standard.